eclair-core/src/main/scala/fr/acinq/eclair/DBChecker.scala

```diff
@@ -16,31 +16,35 @@
 
 package fr.acinq.eclair
 
+import fr.acinq.eclair.channel.Commitments
 import grizzled.slf4j.Logging
 
 import scala.util.{Failure, Success, Try}
 
-object DBCompatChecker extends Logging {
+object DBChecker extends Logging {
 
   /**
-    * Tests if the channels data in the DB are compatible with the current version of eclair; throws an exception if incompatible.
-    *
-    * @param nodeParams
-    */
-  def checkDBCompatibility(nodeParams: NodeParams): Unit =
+   * Tests if the channels data in the DB are compatible with the current version of eclair; throws an exception if incompatible.
+   *
+   * @param nodeParams
+   */
+  def checkChannelsDB(nodeParams: NodeParams): Unit = {
+    logger.info("listLocalChannels()")
```

Comment:
nit: why that log line?

Reply:
removed in https://github.com/ACINQ/eclair/commit/9444d2bd3993cf46ab54b6b4e77861acc088f522